documentation/modules/rn-2.5.adoc

```diff
@@ -241,57 +246,71 @@ spec:
 
 This issue is resolved in {project-short} {project-version}, PVCs and PVs are deleted when deleting migrated VM.link:https://issues.redhat.com/browse/MTV-492[(MTV-492)]
 
 .PVC deletion hangs after archiving and deleting migration plan
 
-In previous releases of {project-short}, when a migration failed, its PVCs and PVs were not deleted as expected when its migration plan was archived and deleted.
+In earlier releases of {project-short}, when a migration failed, its PVCs and PVs were not deleted as expected when its migration plan was archived and deleted.
 
 This issue is resolved in {project-short} {project-version}, PVCs are deleted when archiving and deleting migration plan.link:https://issues.redhat.com/browse/MTV-493[(MTV-493)]
 
-.VM with multiple disks may boot from non-bootable disk after migration
+.VM with multiple disks can boot from a non-bootable disk after migration
 
-In previous releases of {project-short}, VM with multiple disks that were migrated might not have been able to boot on the target {ocp} cluster.
+In earlier releases of {project-short}, VM with multiple disks that were migrated might not have been able to boot on the target {ocp} cluster.
 
 This issue is resolved in {project-short} {project-version}, VM with multiple disks that are migrated are able to boot on the target {ocp} cluster. link:https://issues.redhat.com/browse/MTV-433[(MTV-433)]
 
-
 .Transfer network not taken into account for cold migrations from vSphere
+
 In {project-short} releases 2.4.0-2.5.3, cold migrations from vSphere to the local cluster on which {project-short} was deployed did not take a specified transfer network into account. This issue is resolved in  {project-short} 2.5.4.  link:https://issues.redhat.com/browse/MTV-846[(MTV-846)]
 
-For a complete list of all resolved issues in this release, see the list of link:https://issues.redhat.com/issues/?filter=12424644[Resolved Issues] in Jira.
+.Fix migration of VMs with multi-boot guest operating system from vSphere 
+
+In {project-short} 2.5.6, the virt-v2v arguments include `–root first`, which mitigates an issue with multi-boot VMs where the pod fails. This is a fix for a regression that was introduced in {project-short} 2.4, in which the '--root' argument was dropped. Link:https://issues.redhat.com/browse/MTV-968[(MTV-968)]
+
+.Errors logged in populator pods are improved
+
+In earlier releases of {project-short} {project-version}, populator pods were always restarted on failure. This made it difficult to gather the logs from the failed pods. In {project-short} 2.5.3, the number of restarts of populator pods is limited to three times. On the third and final time, the populator pod remains in the fail status and its logs can then be easily gathered by `must-gather` and by `forklift-controller` to know this step has failed. link:https://issues.redhat.com/browse/MTV-818[(MTV-818)]
+
+.Support deployment on OpenShift 4.15
```

Comment:
we have this one twice now (it already appears in line 39)
here we should rather document the CVE that is mentioned below

Reply:
See text from CVE.
I removed some details from the summary. Let me know if you want all the details in Resolve issues.